app/src/main/java/net/squanchy/home/CurrentEventSnackbarService.java

```diff
@@ -0,0 +1,61 @@
+package net.squanchy.home;
+
+import android.support.design.widget.BaseTransientBottomBar;
+import android.support.design.widget.Snackbar;
+import android.view.View;
+
+import java.util.Locale;
+
+import net.squanchy.R;
+import net.squanchy.schedule.domain.view.Event;
+import net.squanchy.service.firebase.FirebaseAuthService;
+import net.squanchy.service.repository.EventRepository;
+import net.squanchy.support.system.CurrentTime;
+
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+import io.reactivex.Maybe;
+import io.reactivex.Observable;
+
+public class CurrentEventSnackbarService {
```

Comment:
❌  Can we just call it `CurrentEventService` and move out all the `Snackbar` stuff? It **must** not have any notions of snackbars. That should just be how you use its data later on, in the activity. Let's keep the UI and the business logic separate.